hack/boilerplate/boilerplate.BUILD.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 The Cockroach Authors
```

Comment:
@chrisseto all of these headers are matching Google coding standards.  If you have another standard that CRL uses let me know.

Reply:
Looks good to me based off of a spot check from the db repo. @johnrk do you know anything about the licensing expectations for the operator?

Reply:
@chrisseto the licensing expectation is Apache2, are you asking about the wording in the header file?

Reply:
Just wanted to verify that we're going with Apache2 rather than BSL or a mix, I had thought about it previously!

I guess bsl doesn't really make sense here...

Reply:
@johnrk ??  Who can answer this?  I do not want to update all of the headers incorrectly.

Reply:
cc @JuanLeon1 , do you have someone else in mind to review this?

Reply:
Apache2 is the right license to use here. @chrislovecnm the format you're using here is exactly the one we used previously on CRDB.

Reply:
Thanks @petermattis.